src/tzfile.h

```diff
@@ -22,36 +22,35 @@
 */
 
 #ifndef TZDIR
-#define TZDIR	"/usr/share/zoneinfo" /* Time zone object file directory */
-#endif /* !defined TZDIR */
+#define TZDIR "/usr/share/zoneinfo" /* Time zone object file directory */
```

Comment:
This file is copied from another project (https://github.com/eggert/tz/blob/master/tzfile.h).  We shouldn't be reformatting it.

Reply:
Reverted.